src/components/IOUConfirmationList.js

```diff
@@ -108,15 +112,22 @@ class IOUConfirmationList extends Component {
         if (this.props.hasMultipleParticipants) {
             const formattedMyPersonalDetails = getIOUConfirmationOptionsFromMyPersonalDetail(
                 this.props.myPersonalDetails,
-
-                // Convert from cent to bigger form
-                // USD is temporary and there must be support for other currencies in the future
-                `$${this.calculateAmount(true) / 100}`,
+                this.props.translate('iou.request',
+                    {
+                        amount: this.props.numberFormat(this.calculateAmount(true) / 100, {
+                            style: 'currency',
+                            currency: this.props.selectedCurrency.currencyCode,
+                        }),
+                    }),
             );
 
-            // Cents is temporary and there must be support for other currencies in the future
             const formattedParticipants = getIOUConfirmationOptionsFromParticipants(this.props.participants,
-                `$${this.calculateAmount() / 100}`);
+                this.props.translate('iou.request', {
```

Comment:
This is incorrect, we just need the currency symbol, not the `Request [currencySymbol]` string:

![Screenshot 2021-05-20 at 12 30 06](https://user-images.githubusercontent.com/10736861/118971609-8ccdce00-b967-11eb-8d18-ef7a6df333ab.png)